jsapp/scss/components/_kobo.modal.scss

```diff
@@ -707,6 +711,12 @@ $z-modal-x: 10;
           &:focus {
             background: $kobo-white;
           }
+          // If the sibling spans more lines than its translation, the
+          // .translation background color takes over the rest of the space.
+          // This looks ugly, so we change text color to indicate locking instead
+          &:disabled {
+            color: $kobo-midgray;
+          }
         }
```

Comment:
I have a proposition:

```patch
diff --git a/jsapp/scss/components/_kobo.modal.scss b/jsapp/scss/components/_kobo.modal.scss
index 90e5039d357ba28185d921b212f1e8ca927f5361..688c04975e920dc2aabce81f6473640564306a5a 100644
--- a/jsapp/scss/components/_kobo.modal.scss
+++ b/jsapp/scss/components/_kobo.modal.scss
@@ -698,16 +698,17 @@ $z-modal-x: 10;
         min-width: 70%;
       }
       .translation {
-        background: $kobo-green-light;
         color: $kobo-darkgray;
-        padding: 2px;
+        padding: 0;
+
         textarea {
+          background: $kobo-green-light;
           display: block;
-          padding: 8px;
+          padding: 10px;
           border: none;
-          background: transparent;
           width: 100%;
           color: $kobo-black;
+          height: 100% !important;
           &:focus {
             background: $kobo-white;
           }
@@ -716,6 +717,7 @@ $z-modal-x: 10;
           // This looks ugly, so we change text color to indicate locking instead
           &:disabled {
             color: $kobo-midgray;
+            background-color: $kobo-whitegray;
           }
         }
       }

```

Reply:
This way we avoid this weird gray on green and take over the visal side. The downside is having a thin white border around green cells, but it's better anyways :)

Reply:
Also this comment:
```scss
// If the sibling spans more lines than its translation, the
// .translation background color takes over the rest of the space.
// This looks ugly, so we change text color to indicate locking instead
```
Is no longer needed, as `height: 100% !important;` hack override fixes this